deployer/src/emails.py

```diff
@@ -119,10 +118,7 @@ def add(config_name, config_dir=None, emails_to_add=None):
     _commit_push(config_name, 'Update', config_dir)
```

Comment:
Would like to have a add commit :)

Reply:
Not sure to get this one :)

Reply:
Just the message of the commit. Instead of having `Update`